src/main/java/uk/gov/cshr/report/repository/RegisteredLearnerRepository.java

```diff
@@ -18,6 +18,14 @@ public interface RegisteredLearnerRepository extends CrudRepository<RegisteredLe
     """)
     int updateEmail(String uid, String email, ZonedDateTime updatedTimestamp);
 
+    @Modifying(flushAutomatically = true, clearAutomatically = true)
+    @Query("""
+        UPDATE RegisteredLearner rl
+        SET rl.organisationId = null, rl.organisationName = null, rl.updatedTimestamp = :updatedTimestamp
+        WHERE rl.organisationId = :organisationId
+    """)
+    int deleteOrganisation(Long organisationId, ZonedDateTime updatedTimestamp);
```

Comment:
Does `organisationId` not need to be a list so that it deletes all of the child organisations too?